Metallum/Classes/Model/Band.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Bands.swift
+//  Metallum
+//
+//  Created by AppleTree on 3/3/16.
+//  Copyright © 2016 OlhaF. All rights reserved.
+//
+
+import Foundation
+
+final class Band {
+    let name: String?
+    var shortDescription: String?
+    let formationYear: NSNumber?
```

Comment:
If some filed is required (in entity on server) do not use optional but assign default value (0 or "")
In JSON it is NSNumber but due to Interoperability you can assign Int variable to NSNumber variable and vice versa without any effort